src/include/nodes/primnodes.h

```diff
@@ -504,6 +504,17 @@ typedef struct FuncExpr
 	int			location;		/* token location, or -1 if unknown */
 } FuncExpr;
 
+
+/*
+ * FuncDefault - expression node for a function's  default argument
+ */
+typedef struct FuncDefault
```

Comment:
If it is only used by T-SQL, what about `TsqlFuncDefault`?

Reply:
It is right that currently we are only using this node for PL/tsql functions but I am thinking to keep this node and logic very generic so that it can be used for any kind of function.